naff/models/discord/role.py

```diff
@@ -220,3 +220,32 @@ async def move(self, position: int, reason: str | Missing = MISSING) -> "Role":
             self._guild_id, [{"id": self.id, "position": position}], reason
         )
         return self
+
+    # START: fail if not all permissions are manageable
```

Comment:
Why have you left these strange comments?

Reply:
because it are two implementations and one of them will be removed

they are just temporary markers